app/components/README.md

```diff
@@ -0,0 +1,7 @@
+# Components
```

Comment:
WDYT about linking to these from our `docs/frontend.md`? Like a little index/table of contents?

Reply:
> WDYT about linking to these from our `docs/frontend.md`? Like a little index/table of contents?

I think it's a good idea 👍 (I also think it might be hard to enforce for future documentation)

Reply:
> WDYT about linking to these from our `docs/frontend.md`? Like a little index/table of contents?

Added in 082a681a3.